src/lib.rs

```diff
@@ -482,6 +482,11 @@ pub struct WindowAttributes {
     /// [iOS only] Enable multitouch,
     /// see [multipleTouchEnabled](https://developer.apple.com/documentation/uikit/uiview/1622519-multipletouchenabled)
     pub multitouch: bool,
+
+    /// Whether the window should be blurry. This is similar to transparency.
```

Comment:
"blurry" has a pretty strong negative connotation. "blurred" sounds more neutral, but that has associations with input focus... "the window should have a blur effect" probably sounds the coolest, and then the second sentence should go on to explain things more.